slasher/detection/attestations/spanner.go

```diff
@@ -92,12 +97,38 @@ func (s *SpanDetector) DetectSlashingForValidator(
 			}, nil
 		}
 	}
+
+	if sp := s.spans[targetEpoch%numSpans]; sp != nil {
```

Comment:
`sp, ok := s.spans[targetEpoch%numSpans]`

Use `if !ok` ?

Reply:
This is an array, not a map